### ISSUE TITLE
comment:null leads to errors with style-dictionary

### ISSUE THREAD

**Issue by MoritzStefaner** [lukasoppermann/design-tokens#75]
Hi,

thanks for this great plugin! 

One thing I discovered in the latest version:

Missing comments are exported as `comment: null`, which leads to errors when using style-dictionary (I am using 2.10.3) to transform the styles. Changing these to `comment: ""`or removing them altogether leads to a working build.

**Comment by lukasoppermann** [comment]
Hey @MoritzStefaner,

thank you very much for raising this issue. I am working on it and will let you know once it is fixed.

**Comment by MoritzStefaner** [comment]
Ah, interesting. Here's my config:

```
{
  "source": ["./in/**/*.json"],
  "platforms": {
    "scss": {
      "transformGroup": "scss",
      "buildPath": "out/",
      "files": [
        {
          "destination": "theme.scss",
          "format": "scss/variables"
        }
      ]
    },
    "css": {
      "transformGroup": "css",
      "buildPath": "out/",
      "files": [
        {
          "destination": "theme.css",
          "format": "css/variables"
        }
      ]
    },
    "js": {
      "transformGroup": "js",
      "buildPath": "out/",
      "files": [
        {
          "destination": "theme.js",
          "format": "javascript/es6"
        }
      ]
    },
    "json": {
      "transformGroup": "js",
      "buildPath": "out/",
      "files": [
        {
          "destination": "theme.json",
          "format": "json/nested"
        }
      ]
    },
    "scss-kit": {
      "transformGroup": "scss",
      "buildPath": "../www-svelte-kit/static/styles/",
      "files": [
        {
          "destination": "theme.scss",
          "format": "scss/variables"
        }
      ]
    },
    "css-kit": {
      "transformGroup": "css",
      "buildPath": "../www-svelte-kit/static/styles/",
      "files": [
        {
          "destination": "theme.css",
          "format": "css/variables"
        }
      ]
    },
    "js-kit": {
      "transformGroup": "js",
      "buildPath": "../www-svelte-kit/static/styles/",
      "files": [
        {
          "destination": "theme.js",
          "format": "javascript/es6"
        }
      ]
    },
    "json-kit": {
      "transformGroup": "js",
      "buildPath": "../www-svelte-kit/static/styles/",
      "files": [
        {
          "destination": "theme.json",
          "format": "json/nested"
        }
      ]
    }
  }
}
```

**Comment by lukasoppermann** [comment]
Nice, I will try this. I created a PR that should fix it by removing empty comments. I will try to test and merge it tomorrow.

**Comment by lukasoppermann** [comment]
Hey, I just published version 2.1.3 which changes the behavior to avoid any non-string comments

**Comment by MoritzStefaner** [comment]
Sweet, thanks! Can confirm my issues are resolved.
